generic/uname.c

```diff
@@ -67,7 +67,7 @@ static void parseOSRelease(char* buffer, size_t bufferLen) {
    snprintf(buffer, bufferLen, "%s%s%s", name[0] ? name : "", name[0] && version[0] ? " " : "", version);
 }
 
-char* Generic_uname(void) {
+char* Generic_unameRelease(releaseFunction platform_release) {
```

Comment:
Not quite sure about the naming of the argument (and its type).

I think `Platform_FetchReleaseFunction` and `fetchRelease` might be better choices.

Reply:
Yep, OK, will adjust that, thanks.